internals/schema_migration.py

```diff
@@ -1,175 +0,0 @@
-# Copyright 2022 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License")
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import logging
-from google.cloud import ndb  # type: ignore
-
-from framework.basehandlers import FlaskHandler
-from internals import approval_defs
-from internals.core_models import FeatureEntry, Stage
-from internals.legacy_models import Feature
-from internals.review_models import Gate, Vote
-from internals.core_enums import *
-
-
-class EvaluateGateStatus(FlaskHandler):
```

Comment:
We might still need this if we ever detect that the gate status has gotten out of sync with the votes on that gate.  Maybe keep it as-is or rename the file to something like "batch maintenance".

Reply:
I agree, and I definitely think we'll need to write scripts like this in the future. I've renamed the file to `maintenance_scripts.py` and changed the paths to reflect this in `main.py`. If you would prefer a different naming convention, I can update them!